.github/workflows/linux-qe-template.yml

```diff
@@ -131,25 +142,65 @@ jobs:
             -v ${PWD}:/data:z \
             -v ${PWD}/crc:/opt/crc-support/crc:z \
             quay.io/crc-org/ci-crc-support:v2.0.0-dev-linux crc-support/run.sh \
-             -targetPath "/root/crc-support" \
+             -targetPath "/home/crctest/crc-support" \
              -install 'true' \
              -aName 'crc' \
              -freshEnv 'false' \
              -download 'false' 
           podman logs -f crc-linux-install-${{inputs.qe-type}}-${{inputs.preset}}
 
+          # Download arm64 bundle in the reserved machine
+          echo "Start download bundle on reserved machine"
+          
+          if [[ "${{inputs.preset}}" == "microshift" ]]; then
+            bundle_url="https://crcqe-asia.s3.ap-south-1.amazonaws.com/bundles/microshift/4.17.7-arm64"
```

Comment:
Can you pick the bundle version from Makefile? Otherwise we will need to change the version each time